dts/arm/Makefile

```diff
@@ -1,6 +1,6 @@
 ifeq ($(CONFIG_HAS_DTS),y)
 dtb-$(CONFIG_BOARD_DISCO_L475_IOT1)	=	disco_l475_iot1.dts_compiled
-dtb-$(CONFIG_BOARD_FRDM_K64F)	=	frdm_k64f.dts_compiled
+dtb-$(CONFIG_BOARD_FRDM_K64F)		=	frdm_k64f.dts_compiled
```

Comment:
> +dtb-$(CONFIG_BOARD_FRDM_K64F)

Yep, that's the golden rule of making changes - avoid any unrelated ones, the only outcome of these can be increased review time, because reviewers will need to exert more effort to filter important changes from noise. Some smart reviewers may even put such reviews at the end of their queue ;-). Changes like that should go into separate PRs, ideally merged in minutes (though we aren't yet there).